neofetch

```diff
@@ -188,6 +188,12 @@ getdistro() {
                 *) codename="Mac OS X" ;;
             esac
             distro="$codename $osx_version $osx_build"
+
+            case "$distro_shorthand" in
+                "on")   distro="${distro/ ${osx_build}}" ;;
+                "tiny") distro="${distro/${codename}/Mac OS X}"
```

Comment:
Change the `"tiny"` case value to this:

```bash
"tiny")
    case "$osx_version" in
        "10.4"*|"10.5"*|"10.6"*|"10.7"*)
            distro="${distro/${codename}/Mac OS X}" ;;
        "10.8"*|"10.9"*|"10.10"*|"10.11"*)
            distro="${distro/${codename}/OS X}" ;;
        "10.12"*)
            distro="${distro/${codename}/macOS}" ;;
    esac
    distro="${distro/ ${osx_build}}"
;;
```

Otherwise we're always printing "Mac OS X" regardless of version, it should be "OS X" for 10.8 - 10.11 and "macOS" for 10.12 and beyond